lib/mongo-cache.js

```diff
@@ -44,7 +44,7 @@ function makeTileName(req) {
   return req.path + queryString;
 }
 
-function setupQuery(survey, tile) {
+function setupQuery(survey, tile, deleted) {
```

Comment:
Tracking this and some further cleanup in #136. Don't want to futz with the code just yet if it's minor, so we can get the performance change out there.